decidim/decidim
# Fix edit form in intitiatives

#### :tophat: What? Why?
This PR aims to fix the edit initiative form that is currently not handling the translation field. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10723
- Related to #10006
- Fixes #9923

#### Testing
1. Create an initiative 
2. See the fill in data step
3. Check the edit initiative step 
4. See the Text appears like `{"en"=> "The text you've entered"}` in title and description fields
5. Apply patch 
6. See the input value changed to the previously entered value. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
